### PR TITLE
Fix: swap pickup notes for rider_notes on campaign singup.

### PR DESCRIPTION
**Pull request by teesloane**
Fixes #340

### CODE REVIEW

File: lib/bike_brigade_web/live/campaign_signup_live/show.html.heex [hunk 1]
```diff
@@ -23,8 +23,8 @@
     <:col :let={task} label="Dropoff Neighbourhood">
       <%= Locations.neighborhood(task.dropoff_location) %>
     </:col>
-    <:col :let={task} label="Notes"><%= task.delivery_status_notes %>
```
Comment (teesloane) [inline lib/bike_brigade_web/live/campaign_signup_live/show.html.heex:26]: While swapping this, I noticed that for some reasons I added the delivery_status_notes into this html table. 🧹

**Comment by mveytsman** [comment]
Just to be clear we haven't added any form to fill them in yet right?

**Comment by teesloane** [comment]
> Just to be clear we haven't added any form to fill them in yet right?

I did actually add it to the task form. I should have added a screenshot:

CleanShot 2024-04-28 at 16 02 45@2x

**Comment by mveytsman** [comment]
Oh sorry, I missed the html bits. 

To be honest, I'm not 100% sure on this: I think the issue that this has PII and should be hidden from the public page should be addressed.

I think before adding a separate field we should iterate on clear instructions and train dispatchers e.g. I notice there's a todo to rename the field from rider notes to delivery instructions.

What if we make a PR to hide it and then get some feedbakc on slack on copy for adding a second field?

**Comment by teesloane** [comment]
Sure, I'll make a PR to hide the delivery notes for now and then we can circle back to this when necessary :)

**Pull request closed without merging**